README.md

```diff
@@ -1,11 +1,40 @@
-This repository contains Theia extensions related to C/C++ development.
+<div align='center'>
+<br />
+<img src='./logo/theia.svg' alt='theia logo' width='125'>
 
-The `browser-app` and `electron-app` directories contain examples of Theia-based applications containing these extensions.
+<h2>ECLIPSE THEIA - C/C++ EXTENSION</h2>
 
-To build and run the browser application:
+<br />
```

Comment:
Why the `<br />` and not just the following?

```html
<div align='center'>
    <img src='./logo/theia.svg' alt='theia logo' width='125'>
    <h2>ECLIPSE THEIA - C/C++ EXTENSIONS</h2>
</div>
```

Reply:
I wanted to have the appropriate spacing (other theia repos with similar headers do the same)
- https://github.com/theia-ide/generator-theia-extension